common/strings/index.ts

```diff
@@ -13,3 +13,7 @@ export function timingSafeEqual(a: string, b: string) {
 
   return result === 0;
 }
+
```

Comment:
Sorry but:
a) this is not tied in any way to open-source code
b) doesn't belong in the common libraries as it's dataform (web) specific!